static/app/views/issueDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -192,6 +193,11 @@ function GroupEventDetails(props: GroupEventDetailsProps) {
                     project={project}
                   />
                 )}
+                {eventWithMeta && (
```

Comment:
See the lines above and you can notice that `issueTypeConfig.stats.enabled` would prevent issues without stats from rendering.
<img width="367" alt="image" src="https://github.com/getsentry/sentry/assets/44410/278c1dad-cef1-4937-8b7b-2502ec4c2d42">